doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "dB, dBm, dBi, and dBd"
+section: Additional Information
+description: ""
+weight:
+---
+
+In this chapter we learn about some units in brief that are used to measure the performance of transceivers (gateways and end devices) and antennas.
+
+## dB (decibel)
+
+The decibel can be used to express the ratio of two physical quantities such as power, sound intensity, sound pressure, voltage, and current on a logarithmic scale. In LoRaWAN we use decibel to express the ratio between two power levels usually given in Watt (W) or milliwatt (mW).
+
+The power ratio can be expressed in decibels using the formula, 
+
+**dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>)**
```

Comment:
I don't like the way of expressing stuff here. 10log_10(P_out/P_in) is not equal to 1 dB. This is equal to their ratio in dB. You can introduce a new mark for ratio, for example N, and say that this is equal to N_dB (i.e. N in dB, because N doesn't have to be expressed in dB).

Reply:
> I don't like the way of expressing stuff here. 10log_10(P_out/P_in) is not equal to 1 dB.

What's wrong with this formula?

Here is a good example by Massachusetts Institute of Technology (MIT):
![image](https://user-images.githubusercontent.com/36432168/156375788-94da8a6d-cc09-46f7-af5b-7fe7919060e4.png)

Reply:
Nothing is wrong with the formula, the notation is confusing. Why would you take "dB" as name for power gain when the unit is also named dB? It doesn't make sense to me.
Then you get this:

![Screenshot_2022-03-02-16-03-49-145_com google android apps pdfviewer](https://user-images.githubusercontent.com/65215579/156389000-f29daee4-9099-4c35-b649-d9c2f7685664.jpg)

Doesn't it look weird to you to write dB = 20 dB? 

Here's an example of what I think is more convenient:

![Screenshot_2022-03-02-16-03-57-843_com android chrome](https://user-images.githubusercontent.com/65215579/156388943-f096f739-ea9f-45ec-b07f-ab7692dc2542.jpg)